package.json

```diff
@@ -1,82 +1,103 @@
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
-    "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-2": "^6.24.1",
-    "commander": "^2.16.0",
+    "@babel/core": "^7.8.3",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-decorators": "^7.8.3",
+    "@babel/plugin-proposal-export-default-from": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.8.3",
+    "@babel/plugin-transform-arrow-functions": "^7.8.3",
+    "@babel/plugin-transform-async-to-generator": "^7.8.3",
+    "@babel/plugin-transform-classes": "^7.8.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.8.3",
+    "@babel/plugin-transform-object-assign": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/polyfill": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
+    "@babel/runtime": "^7.8.3",
+    "autoprefixer": "^9.7.4",
+    "babel-loader": "^8.0.6",
+    "commander": "^4.1.0",
+    "config": "^3.2.5",
+    "core-js": "^3.6.4",
     "css-flip": "^0.5.0",
-    "css-loader": "^1.0.0",
-    "cssnano": "^4.0.0",
-    "csso-cli": "^1.0.0",
-    "file-loader": "^1.1.11",
-    "fs": "0.0.2",
+    "css-loader": "^3.4.2",
+    "cssnano": "^4.1.10",
+    "csso-cli": "^3.0.0",
+    "file-loader": "^5.0.2",
+    "html-webpack-plugin": "^3.2.0",
     "ignore-assets-webpack-plugin": "^2.0.1",
     "import-glob-loader": "1.1.0",
+    "node-sass": "^4.13.1",
+    "node-sass-glob-importer": "^5.3.2",
     "path": "^0.12.7",
-    "postcss-cli": "^5.0.0",
+    "postcss-cli": "^7.1.0",
     "postcss-loader": "^3.0.0",
-    "react": "16.8.0",
-    "react-dom": "16.8.0",
-    "sass": "^1.23.7",
-    "sass-loader": "^8.0.0",
-    "style-loader": "^0.21.0",
-    "stylelint": "9.3.0",
-    "vue": "2.6.10",
-    "vue-loader": "15.2.4",
-    "vue-style-loader": "4.1.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "sass": "^1.24.5",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "stylelint": "^13.0.0",
+    "vue": "^2.6.11",
+    "vue-loader": "^15.8.3",
+    "vue-style-loader": "^4.1.2",
     "vue-svg-loader": "^0.15.0",
-    "vue-template-compiler": "2.6.10",
-    "vuex": "3.1.1",
-    "vuex-persistedstate": "2.5.4",
-    "webpack": "4.33.0",
-    "webpack-cli": "3.3.2",
-    "webpack-dashboard": "3.0.7",
+    "vue-template-compiler": "^2.6.11",
+    "vuex": "^3.1.2",
+    "vuex-persistedstate": "^2.7.0",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10",
+    "webpack-dashboard": "^3.2.0",
+    "webpack-dev-server": "^3.10.1",
     "webpack-import-glob": "^2.0.0",
-    "webpack-merge": "4.2.1",
-    "webpack-notifier": "1.7.0"
-  }
+    "webpack-merge": "4.2.2",
+    "webpack-notifier": "^1.8.0"
+  },
+  "browserslist": [
```

Comment:
Isn't this redundant since they are already set in the Babel config?

Reply:
Good catch @nomad-mystic but it's the inverse. I neglected to delete targets from our babel config when I consolidated to the package.json. This way our JS and CSS will be targeting the same clients with the same settings.

Reply:
Thanks @nomad-mystic @chrislarrycarl - yes, package.json as canonical is preferred, as the browserslist setting is used by postcss as well as babel